### PR TITLE
blog: "What's new in Svelte" June newsletter

### PR DESCRIPTION
**Pull request by DreaminDani**
Sorry it's a bit last minute, this month! Let me know if I missed anything.

### CODE REVIEW

File: sites/svelte.dev/content/blog/2023-06-01-whats-new-in-svelte-june-2023.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+- The `ResizeObserver` bindings `contentRect`/`contentBoxSize`/`borderBoxSize`/`devicePixelContentBoxSize` are now usable with `bind:`(**3.59.0**, [#8022](https://github.com/sveltejs/svelte/pull/8022))
+
+For all the changes to the Svelte compiler, including unreleased changes, check out the [CHANGELOG](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md).
+
+## What's new in SvelteKit
+- Route-level entry generators allows exporting an entries function from `+page`, `+page.server`, and `+server` files via `export const entries` (**1.16.0**, [Docs](https://kit.svelte.dev/docs/page-options#entries), [#9571](https://github.com/sveltejs/kit/pull/9571))
+- URLs in `<meta>` tags are now crawled to make programmatic social-images much easier (**1.17.0**, [Docs](https://kit.svelte.dev/docs/seo#manual-setup-title-and-meta), [#9900](https://github.com/sveltejs/kit/pull/9900))
+- `data` and `form` have been unshadowed in the `enchance` function - a future deprecation warning has been added for them in `dev` mode (**1.17.0**, [Docs](https://kit.svelte.dev/docs/form-actions#progressive-enhancement-use-enhance), [#9902](https://github.com/sveltejs/kit/pull/9902))
```
Comment (benmccann) [inline sites/svelte.dev/content/blog/2023-06-01-whats-new-in-svelte-june-2023.md:26]:
```suggestion
- `data` and `form` have been unshadowed in the `enhance` function - a future deprecation warning has been added for them in `dev` mode (**1.17.0**, [Docs](https://kit.svelte.dev/docs/form-actions#progressive-enhancement-use-enhance), [#9902](https://github.com/sveltejs/kit/pull/9902))
```

File: sites/svelte.dev/content/blog/2023-06-01-whats-new-in-svelte-june-2023.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+- Link options can now be set to "true" and "false" (**1.19.0**, [Docs](https://kit.svelte.dev/docs/link-options#disabling-options), [#10039](https://github.com/sveltejs/kit/pull/10039))
```
Comment (benmccann) [inline sites/svelte.dev/content/blog/2023-06-01-whats-new-in-svelte-june-2023.md:27]:
```suggestion
- Link options can now be set to `true` and `false` (**1.19.0**, [Docs](https://kit.svelte.dev/docs/link-options#disabling-options), [#10039](https://github.com/sveltejs/kit/pull/10039))
```

**Comment by DreaminDani** [comment]
Thanks for the feedback @benmccann! I committed your suggestions and added a mention of Svelte 4.0 in the upper section. I think this is good to merge :)

File: sites/svelte.dev/content/blog/2023-06-01-whats-new-in-svelte-june-2023.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+- `fullscreenElement` and `visibilityState` bindings are now available for the `<svelte:document>` element (**3.59.0**, [#8507](https://github.com/sveltejs/svelte/pull/8507))
+- The `devicePixelRatio` binding is now available for the `<svelte:window>` element (**3.59.0**, [#8285](https://github.com/sveltejs/svelte/issues/8285))
+- The `ResizeObserver` bindings `contentRect`/`contentBoxSize`/`borderBoxSize`/`devicePixelContentBoxSize` are now usable with `bind:`(**3.59.0**, [#8022](https://github.com/sveltejs/svelte/pull/8022))
+
+For all the changes to the Svelte compiler, including unreleased changes, check out the [CHANGELOG](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md).
+
+## What's new in SvelteKit
+- Route-level entry generators allows exporting an entries function from `+page`, `+page.server`, and `+server` files via `export const entries` (**1.16.0**, [Docs](https://kit.svelte.dev/docs/page-options#entries), [#9571](https://github.com/sveltejs/kit/pull/9571))
```
Comment (gtm-nayan) [inline sites/svelte.dev/content/blog/2023-06-01-whats-new-in-svelte-june-2023.md:27]:
```suggestion
- Route-level entry generators allow exporting an entries function from `+page`, `+page.server`, and `+server` files to supply possible values for params for prerendering (**1.16.0**, [Docs](https://kit.svelte.dev/docs/page-options#entries), [#9571](https://github.com/sveltejs/kit/pull/9571))
```

**Review by dominikg — APPROVED** [review]

**Pull request merged**